content/docs/using-tina-editor.md

```diff
@@ -1,45 +1,43 @@
+Tina is very flexible & extendable, so any Tina site may have a unique experience. As an introduction, we'll cover the out-of-the-box experience that most of our starters implement.
 
-## Entering Edit-mode
+## Entering the CMS
 
-When Tina is initialized on a NextJS site, a "/admin" page is created to allow editors to login.
+When Tina is initialized on a NextJS site, a "/admin" page is created to allow editors to log in and begin to make content changes.
 
-> Note: For demo purposes, some of our starters show a "Edit with Tina" link to the "/admin". This can be removed on production sites.
+![Tina Login Page](/img/tina-login.png)
 
-![tina-tailwind-starter](https://res.cloudinary.com/forestry-demo/image/upload/c_scale,q_49,w_800/v1638554159/tina-io/using-tina/starter.png)
+After successfully logging in, the user will arrive at the CMS.
 
-On the "/admin" page, you will see a "Log In" button.
+## Using the CMS
 
-> Note: When working in local-mode, you will not actually need to log in. In production-mode, only authorized users will be able to enter edit-mode on your site.
+![Tina Editor Dashboard](/img/tina-dashboard.png)
 
-![tina-admin](https://res.cloudinary.com/forestry-demo/image/upload/c_scale,q_50,w_800/v1638554159/tina-io/using-tina/admin.png)
+Depending on how Tina is configured, one or more Collections will appear in the Global Navigation. Clicking on a Collection will take the user to a Document List for that Collection.
 
-Clicking "Log in" will put the user in "edit-mode" and take you back to the homepage.
+![Tina Editor List Page](/img/tina-list-page.png)
 
-## Editing the content
+The table here lists all documents for the chosen Collection. This page also allows the user to create a new document.
 
-Once in edit-mode, you should see a little pencil icon in the bottom left corner.
+If **contextual-editing** has been configured, clicking on a document will navigate the user to that document's preview with the Sidebar open and ready for editing.
 
-Try clicking it and making some changes to the content!
+![Tina Editor Contextual Editing](/img/tina-contextual-editing.png)
 
-![using-tina-sidebar](https://res.cloudinary.com/forestry-demo/image/upload/v1638554818/tina-io/using-tina/sidebar.gif)
+However, if **contextual-editing** is not configured or simply not available for a particular Collection, clicking on a document will present the user with the basic editor.
 
-Once you hit "Save Changes"..
+![Tina Editor Standard Editing](/img/tina-standard-editing.png)
 
-- In "Local Mode", any changes that you make will be saved to the local Markdown/JSON files.
+> Note: You can learn more about how to set up **contextual-editing** for your users [here](/docs/tinacms-context).
 
-![tina-markdown](https://res.cloudinary.com/forestry-demo/image/upload/c_scale,q_49,w_800/v1638554159/tina-io/using-tina/markdown.png)
+Once saved, the changes to the document, whether in contextual-editing or not, will be stored against the files.
```

Comment:
Feels a tad wordy. Maybe this is fine?
```suggestion
Once saved, the changes to the document will be stored against the files.
```

Reply:
Love it.  I was actually removing quite a bit of "wordy-ness" from this yesterday.  Good catch 👍